src/App.jsx

```diff
@@ -1,12 +1,15 @@
 import { Routes, Route } from 'react-router-dom'
 import UserRoutes from './routes/UserRoutes';
+import UserLayout from './layouts/UserLayout';
 // import'./App.css';
 function App() {
 
 
   return (
     <Routes>
-      <Route path="/*" element={<UserRoutes/>} />
+       <Route path="/*" element={<UserLayout />}>
+          <Route path="*" element={<UserRoutes />} />
```

Comment:
why you added this?

Reply:
to keep the layout same for all pages, should i remove userlayout from here and restructure it another way ?